### DISCUSSION TITLE
problem with proxy

### DISCUSSION THREAD

**Discussion by carlosribeiro** [globaleaks/globaleaks-whistleblowing-software#3560] [Q&A]
Hello,
I already realized that it is not recommended to use proxies but based on my infrastructure I really needed to activate globaleaks after a proxy (Nginx Proxy Manager).
Is there any way to configure?

Globaleaks Version: v4.12.5

**Comment by gianlucagilardi** [comment]
I can't see why it should not be possible (I am doing it using traefik as a reverse proxy). Going to be a bit tricky though, given the recent changes in the SSL management in globaleaks; you should search for "SNI" and "SSL passthrough", while maintaining port 80 proxying for let's encprypt.

**Comment by evilaliv3** [comment]
@carlosribeiro have you tried to connect to globaleaks on port 8080 and 8443?

GlobaLeaks tries to bind on 80 and 443 when started as root or alternatively on port 8080 and 8443 when started rootless.

**Reply by carlosribeiro** [reply]
Port 80 is redirect to 443 automatically. Port 8080 and 8443 not exposed external.
